modin/tests/pandas/utils.py

```diff
@@ -1087,7 +1185,11 @@ def eval_io_from_str(csv_str: str, unique_filename: str, **kwargs):
 
 
 def create_test_dfs(
-    *args, post_fn=None, backend=None, **kwargs
+    *args,
+    post_fn=None,
+    backend=None,
+    data_frame_mode=None,
```

Comment:
```suggestion
    df_mode=None,
```
Let's use this everywhere.

Reply:
Done